cli/policies.go

```diff
@@ -84,50 +114,60 @@ var cmdPolicies = []cobra.Command{
 				Offset: uint64(Offset),
 				Limit:  uint64(Limit),
 			}
-			if args[0] == "things" {
-				policies, err := sdk.ListThingsPolicies(pm, args[1])
+			switch args[0] {
+			case things:
+				policies, err := sdk.ListThingPolicies(pm, args[1])
 				if err != nil {
 					logError(err)
 					return
 				}
 				logJSON(policies)
 				return
-			}
-			if args[0] == "users" {
-				policies, err := sdk.ListPolicies(pm, args[0])
+			case users:
```

Comment:
Getting Status: Unauthorized: failed to perform authentication over the entity
```bash 
$ ./build/mainflux-cli policies list users  $token`
error: Status: Unauthorized: failed to perform authentication over the entity
```

Reply:
This has been fixed by https://github.com/mainflux/mainflux/pull/1862